core/templates/pages/collection-player-page/collection-player-page.module.ts

```diff
@@ -35,22 +35,22 @@ import { CollectionNodeListComponent } from
   'pages/collection-player-page/collection-node-list/collection-node-list.component';
 import { platformFeatureInitFactory, PlatformFeatureService } from
   'services/platform-feature.service';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
 @NgModule({
   imports: [
     BrowserModule,
+    BrowserAnimationsModule,
```

Comment:
Can we move it to `SharedComponentsModule` if it's being used in all the page modules? (not sure!)

Reply:
No, we need BrowserModule and BrowserAnimationsModule on oppia-root.module. Shared components module is lazily loaded and angular doesnot allow BrowserModule and BrowserAnimationsModule registered with more than one module on a page.
After completion of migration (both angular and angular router). BrowserModule/ BrowserAnimationsModule will registered only at one file (oppia-root.module.ts).

Reply:
Thanks for the explanation!